WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -38,21 +38,22 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v18.7-whats-new"
+msgctxt "v18.8-whats-new"
 msgid ""
-"Two-step authentication now comes with extra security. You'll need to unlock your device to approve or deny notifications.\n"
-"We expanded the “Link To” setting for the image block to include “Media File” and “Attachment Page” options alongside the preexisting “Custom URL” option.\n"
-"We fixed an issue that made the interface crash when users copied and pasted lists from Microsoft Word.\n"
+"Good news, everyone—we fixed app crashes caused by uploading images while offline, deleting media, or seeing details for "Like" notifications.\n"
```

Comment:
I'm kind of surprised this passed linting with the inline `"` tbh.

Reply:
Good point, it actually does not pass local, manual linting on my Mac indeed:

```
$ xgettext -o /dev/null WordPress/Resources/AppStoreStrings.po

WordPress/Resources/AppStoreStrings.po:39: keyword "Like" unknown
WordPress/Resources/AppStoreStrings.po:38: missing 'msgstr' section
WordPress/Resources/AppStoreStrings.po:39:128: syntax error
WordPress/Resources/AppStoreStrings.po:43: keyword "Link" unknown
WordPress/Resources/AppStoreStrings.po:43: keyword "To" unknown
WordPress/Resources/AppStoreStrings.po:46: keyword "Edit" unknown
xgettext: found 6 fatal errors
```

I don't think we have linting set up on CI for this repo (hence it not warning us about it)?